markymark-core/src/test/java/com/m2mobi/markymark/MarkyMarkBuildTest.java

```diff
@@ -14,26 +41,27 @@
     @Test
     public void buildInstance() {
         MarkyMark.Builder builder = new MarkyMark.Builder();
-        Flavor flavor = mock(Flavor.class);
-        Rule rule = mock(Rule.class);
-        Converter converter = mock(Converter.class);
-        InlineConverter inlineConverter = mock(InlineConverter.class);
+        Flavor flavor = INSTANCE.mock(Flavor.class);
+        Rule rule = INSTANCE.mock(Rule.class);
```

Comment:
Any reason why you stopped statically importing these?

Reply:
Kotlin object are singletons because they are instantiated under the hood. Java can work with them by referencing this singleton instance. However because it is an instance as far as java knows you can't statically import its functions just its instance. It would have been fantastic if I could have statically imported the functions as well but this will have to do for now. Its an interop hack anyway thats got to be removed with time.